Fix windows subsystem documentation (#66)
NTIG-Uppsala/elevhalsa-skylt: documentation.md

- Go to the Control Panel → Programs → Programs and Feautures → Turn Windows features on or off

> - Check the Windows Subsystem for Linux box (commonly found near the bottom) and press OK. Windows will prompt a restart and press restart

Reviewer: ```suggestion
- Check the "Windows Subsystem for Linux" box (commonly found near the bottom) and press OK. Restart the computer when prompted.
```
